### PR TITLE
deprecate fortigate integration

### PR DESCRIPTION
**Pull request by kifeo**
## Proposed change

Deprecate the fortigate integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/34586
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by klaasnicolaas** [comment]
This PR fits better in the current branch. Since you already want to alert people that the integration is going to be deprecated.

Then create a new PR with the target branch next and remove everything from the integration in the docs, since your parent PR also removes everything in the codebase.

File: source/_integrations/fortigate.markdown [hunk 1]
```diff
@@ -12,6 +12,8 @@ ha_domain: fortigate
 
 This is a FortiGate presence sensor based on device detection of the FortiGate API
 
+This integration will be deprecated in order to only use the fortios integration
```
Comment (klaasnicolaas) [inline source/_integrations/fortigate.markdown]: Please put this in a note and maybe a link to the fortios integration

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/fortigate.markdown (hunk 1 above)
Comment (kifeo) [reply source/_integrations/fortigate.markdown]: very good idea, thanks

Comment (klaasnicolaas) [reply source/_integrations/fortigate.markdown]: Can you use this format: https://developers.home-assistant.io/docs/documentation_create_page/#html

Comment (kifeo) [reply source/_integrations/fortigate.markdown]: changed it, thanks

**Review by fabaff — APPROVED** [review]

File: source/_integrations/fortigate.markdown [hunk 2]
```diff
@@ -12,6 +12,10 @@ ha_domain: fortigate
 
 This is a FortiGate presence sensor based on device detection of the FortiGate API
 
+<div class='note warning'>
+This integration will be deprecated. Please use the [FortiOS](/integrations/fortios) integration.
+</div>
```
Comment (frenck) [inline source/_integrations/fortigate.markdown:17]: This is now broken on our website as the HTML & Markdown context isn't switched.
At least one new line is needed after switching between HTML & Markdown.

So after the opening div and before the closing div an empty line needed to be put.

**Pull request merged**